Dockerfile.daemon

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.17 AS builder
+WORKDIR /go/src/github.com/openshift/ingress-node-firewall
+COPY . .
+RUN make daemon
+
+FROM quay.io/centos/centos:stream8
```

Comment:
not sure why u picked this distro and will it allow us to mount bpffs ?
also pls create OCP flavor of this dockerfile so we can get the CI building the ds images

Reply:
it's just the same that the sriov team is using. I hadn't given it a thought, I'll use what you prefer